### DISCUSSION TITLE
(ScreenXpert) Option to control second screen on Zephyrus Duo 16 (GX650RX)

### DISCUSSION THREAD

**Discussion by slashdelta** [seerge/g-helper#650] [General]
Hello,

I have a GX650RX Duo 16 (2022), and I'm loving G-Helper, gets rid of the bloat ArmoryCrate like a boss.

I have wiped my machine multiple times, and yet it always seems to crash with ScreenXpert. It will work flawlessly for about 2 days, then freeze out and slowly display all the UI elements onto both screens. I'll then have to hard reset to get it working normally. But after it comes back, its all good, until it brain-damages out again, roughly 2 days.

I mainly use the second screen for videos playing while working. And the only main thing I would need function on it would be the brightness slider, and maybe changing resolution (but that can be done in Windows Display Settings)

Any thoughts on replacing ScreenXpert aswell?
Anyone else with multi-screen machines experiencing the same problems?

(FYI, if you need a tester for the Duo, let me know)

Cheers,

slashdelta

**Comment by seerge** [comment]
@slashdelta hello, g-helper can fix issues with other 3rd party software (ScreenXpert) anyhow.

**Comment by Provinomico** [comment]
Also a Duo user (2023 model). I've not experienced any problems with ScreenXpert so far. Do you have the Asus optimization service enabled? I've found to be needed on mine for the secondary screen to work properly.

**Reply by slashdelta** [reply]
I've done a clean install, and tested it out for the last 2 days, seems stable enough. Must have been an old config or setting that was stuck. All good now. I've just turned on the Optimization service, I'll see if there is any change with that one, thanks!

Now I'm getting a strange issue where the Laptop Screen settings in G-Helper aren't there, or they are showing my externally connected monitor. So no changing the refresh rate or the multizone lighting. I'll test out more, and if needed, start a new thread on it

Thanks for the responses

slash

**Comment by 833s3y833** [comment]
I've had similar issues, but mine has been the NVIDIA drivers. I have the GX650PY 2023, I have wiped mine using ShredOS and ever since, I haven't been able to install the NVIDIA drivers without getting screen artifacting immediately upon windows sign-in after boot. Followed by a repetitive boot loop. Until I boot into safe-mode and use DDU and completely remove all NVIDIA and armoury crate software. Currently I am using Windows basic drivers (Which really sucks) but i haven't been able to find a solution yet. I can provide the ScreenXpert download that I have if you'd like. You'd think with the price we pay for these machines we wouldn't have to worry about these senseless issues.
